src/lib/routes/history.js

```diff
@@ -0,0 +1,2 @@
+import { createBrowserHistory } from 'history';
+export const history = createBrowserHistory();
```

Comment:
Hmm, should we grab this from gisida instead of having the function here? If we do that then we could set it up as a singleton as well